### PR TITLE
Update azure from master for release 4.13 only

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

File: Testing/ContinuousIntegration/AzurePipelinesMacOSPython.yml [hunk 1]
```diff
@@ -3,28 +3,53 @@ trigger:
...
+      workingDirectory: $(Agent.BuildDirectory)
+    - bash: |
+        set -x
 
         c++ --version
         cmake --version
 
         export BUILD_EXAMPLES=OFF
-        export CTEST_BUILD_CONFIGURATION=MinSizeRel
+        export CTEST_BUILD_CONFIGURATION=Release
```
Comment (dzenanz) [inline Testing/ContinuousIntegration/AzurePipelinesMacOSPython.yml]: Is this change intentional?

Comment (blowekamp) [reply Testing/ContinuousIntegration/AzurePipelinesMacOSPython.yml]: nope!
